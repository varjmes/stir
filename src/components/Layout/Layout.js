import React from 'react'
import { createGlobalStyle } from 'styled-components'

import { Container } from './styles'
import Header from './Header/Header'

const GlobalStyles = createGlobalStyle`
  body {
    background: #ffebef;
  }
`

export default ({ children }) => (
  <Container>
    <GlobalStyles />
    <Header />
    {children}
  </Container>
)
