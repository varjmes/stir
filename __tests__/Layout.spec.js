import React from 'react'
import renderer from 'react-test-renderer'

import Layout from '../src/components/Layout/Layout'

describe('Layout', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Layout />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
