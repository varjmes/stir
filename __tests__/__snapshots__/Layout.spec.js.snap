// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<main
  className="sc-bdVaJa cHrwRm"
>
  <h1
    className="sc-bwzfXH jotfnM"
  >
    stir.
  </h1>
</main>
`;
